data/virtualization/autoyast/host_15.xml.ep

```diff
@@ -0,0 +1,305 @@
+    </mode>
+    <proposals config:type="list"/>
+    <signature-handling>
+      <accept_file_without_checksum config:type="boolean">true</accept_file_without_checksum>
+      <accept_non_trusted_gpg_key config:type="boolean">true</accept_non_trusted_gpg_key>
+      <accept_unknown_gpg_key config:type="boolean">true</accept_unknown_gpg_key>
+      <accept_unsigned_file config:type="boolean">true</accept_unsigned_file>
+      <accept_verification_failed config:type="boolean">true</accept_verification_failed>
+      <import_gpg_key config:type="boolean">true</import_gpg_key>
+    </signature-handling>
+    <storage>
+      <start_multipath config:type="boolean">false</start_multipath>
+    </storage>
+  </general>
+  <kdump>
+    <add_crash_kernel config:type="boolean">true</add_crash_kernel>
+    <crash_kernel>201M,high</crash_kernel>
+    <crash_xen_kernel>201M\&lt;4G</crash_xen_kernel>
+  </kdump>
+  <keyboard>
+    <keymap>english-us</keymap>
+  </keyboard>
+  <language>
+    <language>en_US</language>
+    <languages/>
+  </language>
+  <login_settings/>
+  <networking>
+    <keep_install_network config:type="boolean">true</keep_install_network>
+<interfaces config:type="list">
+  <interface>
+    <device>br0</device>
+    <bootproto>dhcp</bootproto>
+    <bridge>yes</bridge>
+    <bridge_forwarddelay>0</bridge_forwarddelay>
+    <bridge_ports>eth0 eth1</bridge_ports>
+    <bridge_stp>off</bridge_stp>
+    <startmode>auto</startmode>
+  </interface>
+  <interface>
+    <device>eth0</device>
+    <bootproto>none</bootproto>
+    <startmode>hotplug</startmode>
+  </interface>
+  <interface>
+    <device>eth1</device>
+    <bootproto>none</bootproto>
+    <startmode>hotplug</startmode>
+  </interface>
+</interfaces>
+  </networking>
+  <ntp-client>
+    <ntp_policy>auto</ntp_policy>
+    <ntp_servers config:type="list">
+      <ntp_server>
+        <address>europe.pool.ntp.org</address>
+        <iburst config:type="boolean">true</iburst>
+        <offline config:type="boolean">false</offline>
+      </ntp_server>
+    </ntp_servers>
+    <ntp_sync>systemd</ntp_sync>
+  </ntp-client>
+  <partitioning config:type="list">
+    <drive>
+      <!--device>/dev/sda</device-->
+      <disklabel>msdos</disklabel>
+      <enable_snapshots config:type="boolean">false</enable_snapshots>
+      <initialize config:type="boolean">true</initialize>
+      <partitions config:type="list">
+        <partition>
+          <create config:type="boolean">true</create>
+          <create_subvolumes config:type="boolean">true</create_subvolumes>
+          <filesystem config:type="symbol">btrfs</filesystem>
+          <format config:type="boolean">true</format>
+          <mount>/</mount>
+          <mountby config:type="symbol">uuid</mountby>
+          <partition_id config:type="integer">131</partition_id>
+          <partition_nr config:type="integer">1</partition_nr>
+          <resize config:type="boolean">false</resize>
+          <size>400G</size>
```

Comment:
Do all machines have one single disk with size >400G? Besides, there is no mount point given. Why not assign it to /?

Reply:
Different machines have different disk size. The minimum size is 450G so assign 400G to root partition and 20G  to swap. See line 140 for / mount point